contrib/net/http/http.go

```diff
@@ -65,6 +66,9 @@ func WrapHandler(h http.Handler, service, resource string, opts ...Option) http.
 			h.ServeHTTP(w, req)
 			return
 		}
+		if dynamicResource := cfg.resourceNamer(req); dynamicResource != "" {
+			resource = dynamicResource
```

Comment:
```suggestion
		if r := cfg.resourceNamer(req); r != "" {
			resource = r
```
Please see https://github.com/DataDog/dd-trace-go/wiki/Style-guidelines#naming-things if clarification is needed.